app/validators/image_attachments_validator.rb

```diff
@@ -0,0 +1,24 @@
+class ImageAttachmentsValidator < ActiveModel::Validator
+  def validate(record)
+    return unless record.description&.body&.attachments&.any?
```

Comment:
Шось дуже багато `&`, а мало б не бути взагалі. Чого так?

Reply:
Це через той річ текст, ясно.

Reply:
Що буде якщо просто?
```suggestion
    return unless record.description
```

Reply:
Доречі `validates_with` приймає ще `:if` та `:unless`: https://apidock.com/rails/v6.0.0/ActiveModel/Validations/ClassMethods/validates_with

Reply:
Там далі йде виклик body.attachments, якщо їх не перевіряти то буде помилка.
Плюс там важливо перед тим як виконають валідації для зображення виконати перевірку чи там є взагалі вкладення, а не просто текст

Reply:
`body.attachments` може повертати `nil` чи там завжди буде масив/relation?
Якщо масив/relation, то в `.attachments&.any?` нема потреби.

Взагалі такі ланцюжки робити - зазвичай погана практика, а якщо ще й через `&`, то це вже в квадраті погана практика і значить з кодом точно шось не так.